python/CMakeLists.txt

```diff
@@ -2,8 +2,8 @@
 IF (CTYPESGEN_FOUND)
   ADD_CUSTOM_COMMAND(
     OUTPUT GraphQLParser.py
-    COMMAND ctypesgen.py ../c/*.h -o GraphQLParser.py -l graphqlparser -L .. 2>&1 > /dev/null
-    DEPENDS ../c/GraphQLAstForEachConcreteType.h ../c/GraphQLAst.h
+    COMMAND ctypesgen.py ${CMAKE_CURRENT_SOURCE_DIR}/../c/*.h ${CMAKE_CURRENT_BINARY_DIR}/../c/*.h -o ${CMAKE_CURRENT_SOURCE_DIR}/GraphQLParser.py -I ${CMAKE_CURRENT_SOURCE_DIR}/.. -I ${CMAKE_CURRENT_BINARY_DIR}/.. -l graphqlparser -L ${CMAKE_CURRENT_BINARY_DIR}/.. 2>&1 > /dev/null
+    DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/../c/GraphQLAstForEachConcreteType.h ${CMAKE_CURRENT_BINARY_DIR}/../c/GraphQLAst.h
```

Comment:
you probably want to list out all the headers in the depends and command explicitly if you want correct dependencies.